pkg/storage/migrate/registry.go

```diff
@@ -0,0 +1,131 @@
+package migrate
```

Comment:
Since this is a publicly exported package of the openfga project, can we add some godoc to the signatures introduced here?

If I was a new contributor on this project and wanted to contribute a new SQL storage adapter, it'd be nice to have some godoc.